zh_TW/LC_MESSAGES/django.po

```diff
@@ -7758,9 +7758,11 @@ msgid ""
 "                This value may contain only letters, numbers, and @/./+/-/_ characters.\n"
 "                "
 msgstr ""
-"請注意使用者名稱會是公開的。\n"
-"這是必填欄位，不能與其他人重複，且必須少於 150 字元。\n"
-"欄位內容僅能包含拉丁字母、數字、@ . + - _ 等符號。"
+"\n"
+"                請注意使用者名稱會是公開的。\n"
+"                這是必填欄位，不能與其他人重複，且必須少於 150 字元。\n"
+"                欄位內容僅能包含拉丁字母、數字、@ . + - _ 等符號."
```

Comment:
Same here. I think we should keep the same period-like character that was used in the translation.
```
"                欄位內容僅能包含拉丁字母、數字、@ . + - _ 等符號。"
```